content/docs/handling-events.md

```diff
@@ -54,19 +54,19 @@ function ActionLink() {
 }
 ```
 
-Here, `e` is a synthetic event. React defines these synthetic events according to the [W3C spec](https://www.w3.org/TR/DOM-Level-3-Events/), so you don't need to worry about cross-browser compatibility. See the [`SyntheticEvent`](/docs/events.html) reference guide to learn more.
+Aquí, `e` es un evento sintético. React define estos eventos sintéticos acorde a las [especificaciones W3C](https://www.w3.org/TR/DOM-Level-3-Events/), entonces no debes preocuparte por la compatibilidad a tráves de los navegadores. Mira la guía de referencia [`SyntheticEvent`](/docs/events.html) para aprender más.
```

Comment:
What do you think about `deberías` instead of `debes`

Reply:
I think `deberías` sounds like some times you won't need to worry, but in some cases you will. In that case it would be `... you shouldn't need to worry...`. LMK what do you think about this one